### ISSUE TITLE
Version Notes are not appearing for some links

### ISSUE THREAD

**Issue by luislobo** [balderdashy/sails-docs#678]
Version Notes in http://sailsjs.org/support link to this versions, and the page appears as empty.
- http://sailsjs.org/version-notes/0point12pointx
- http://sailsjs.org/version-notes/0point11pointx
- http://sailsjs.org/version-notes/0point10pointx

Also, the version notes index itself looks strange: it groups by minor version, but shows all versions on each group:
![screenshot from 2016-04-19 16-46-19](https://cloud.githubusercontent.com/assets/453120/14653265/9c422c7c-064e-11e6-94ab-8b76f41dca54.png)

**Comment by sailsbot** [comment]
@luislobo: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by luislobo** [comment]
@mikermcneil @rachaelshaw 
This still appears to be unfixed

**Comment by sailsbot** [comment]
@luislobo,@sailsbot: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by luislobo** [comment]
This is still unfixed

**Comment by sailsbot** [comment]
@luislobo,@sailsbot: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by luislobo** [comment]
It is still unfixed @rachaelshaw

**Comment by sailsbot** [comment]
@luislobo,@sailsbot: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by luislobo** [comment]
@mikermcneil this is still outstanding...

**Comment by rachaelshaw** [comment]
@luislobo this should be fixed now. That page was using `_.where`, to build up the sub-lists, which is no longer supported in newer versions of lodash, so I think somehow the wrong version of lodash is being installed when the site gets deployed.  The version notes would look fine locally (even in production mode), but kept getting messed up when I redeployed the site.  I switched it to use `_.filter` instead, which solves this particular issue, but @mikermcneil @particlebanana we should probably figure out wtf is up with the lodash version on the deployed site, so we don't get any more nasty surprises.
